package/firmware/rcw/Makefile

```diff
@@ -9,13 +9,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
 PKG_NAME:=rcw
-PKG_VERSION:=v1.0
+PKG_VERSION:=2017.07
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
-PKG_MIRROR_HASH:=8082979dbdef57e36a3096986aa426ef678c152403eb1dbd132da28f01490b6d
-PKG_SOURCE_URL:=https://github.com/fsl-jyt/rcw.git
-PKG_SOURCE_VERSION:=17e09e19bd5cf94e5ebc84916b4607743c29723e
+PKG_MIRROR_HASH:=8fcaed3bc0230a84863eca333a1cd0fc700db783b236eaa35c1f25060ca85a09
+PKG_SOURCE_URL:=https://github.com/qoriq-open-source/rcw.git
+PKG_SOURCE_VERSION:=6eed8fdfc07920178b70fba199fa20fb42adb20b
 
 PKG_BUILD_DIR=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
 
```

Comment:
This package is licensed under some BSD license and not GPL:
https://github.com/qoriq-open-source/rcw/blob/integration/LICENSE